src/main/java/io/github/nucleuspowered/nucleus/commands/item/RepairCommand.java

```diff
@@ -0,0 +1,78 @@
+import io.github.nucleuspowered.nucleus.internal.annotations.Modules;
+import io.github.nucleuspowered.nucleus.internal.annotations.NoCooldown;
+import io.github.nucleuspowered.nucleus.internal.annotations.NoCost;
+import io.github.nucleuspowered.nucleus.internal.annotations.NoWarmup;
+import io.github.nucleuspowered.nucleus.internal.annotations.RegisterCommand;
+import io.github.nucleuspowered.nucleus.internal.permissions.PermissionInformation;
+import io.github.nucleuspowered.nucleus.internal.permissions.SuggestedLevel;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.CommandSource;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.command.args.GenericArguments;
+import org.spongepowered.api.command.spec.CommandSpec;
+import org.spongepowered.api.data.manipulator.mutable.item.DurabilityData;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.item.inventory.ItemStack;
+import org.spongepowered.api.text.Text;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+@RegisterCommand({"repair", "mend"})
+@Modules(PluginModule.ITEMS)
+@NoCooldown
+@NoWarmup
+@NoCost
+public class RepairCommand extends CommandBase<CommandSource> {
+
+    private final String player = "player";
+
+    @Override
+    public Map<String, PermissionInformation> permissionSuffixesToRegister() {
+        Map<String, PermissionInformation> m = new HashMap<>();
+        m.put("others", new PermissionInformation(Util.getMessageWithFormat("permission.others", this.getAliases()[0]), SuggestedLevel.ADMIN));
+        return m;
+    }
+
+    @Override
+    public CommandSpec createSpec() {
+        return CommandSpec.builder().executor(this).arguments(GenericArguments.optional(GenericArguments.requiringPermission(
+                GenericArguments.onlyOne(GenericArguments.player(Text.of(player))), permissions.getPermissionWithSuffix("others")))).build();
+    }
+
+    @Override
+    public CommandResult executeCommand(CommandSource src, CommandContext args) throws Exception {
+        Optional<Player> opl = this.getUser(Player.class, src, player, args);
+        if (!opl.isPresent()) {
+            return CommandResult.empty();
+        }
+
+        if (opl.get().getItemInHand().isPresent()) {
+            ItemStack stack = opl.get().getItemInHand().get();
+
+            if (stack.get(DurabilityData.class).isPresent()) {
+                DurabilityData durabilityData = stack.get(DurabilityData.class).get();
+                durabilityData.durability().set(durabilityData.durability().getDefault());
+                stack.offer(durabilityData);
+                src.sendMessage(Util.getTextMessageWithFormat("command.repair.success", opl.get().getName()));
+                return CommandResult.success();
+            } else {
+                src.sendMessage(Util.getTextMessageWithFormat("command.repair.error"));
```

Comment:
We're checking for `DurabilityData` - there wasn't an error, the item in the hand is not repairable.